### PR TITLE
Rename Vector Tile source functions and Add GEOJSON attribute to trip

### PR DESCRIPTION
**Pull request by TimVesputi**

### CODE REVIEW

**Comment by TimVesputi** [comment]
TODO: create issue to remove deprecated versions of methods in version 4.0

File: README.md [hunk 1]
```diff
@@ -136,22 +156,54 @@ You can access the urls only by refering to `mobilitybox.vector_tile_source().ti
...
+- [Add a Vector Tile Source to mapbox](https://docs.mapbox.com/mapbox-gl-js/example/vector-source/)
 
 ```js
 mapbox_map.addSource('relevant_route_tiles', window.mobilitybox.relevant_routes_vector_tile_source())
 ```
 
+#### Mobilitybox.transit_map_vector_tile_source()
+(DEPRECATED: Mobilitybox.relevant_routes_vector_tile_source())
+
+In order to show a transit map with paths for different routes on a map it is usefull to transfer them as map tiles. (Gives you only the currently displayed segments.)
```
Comment (rosario-raulin) [inline README.md]: We should explain what the transit map layer is, for instance:

> The transit map layer is a set of lines on a map which represent the typical routes public transit vehicles take. This includes for example buses, trams and subways. This transit map layer might show routes of different importance on different zoom levels.

File: src/index.js [hunk 2]
```diff
@@ -68,6 +68,32 @@ export class Mobilitybox {
       .then(response => new MobilityboxTrip(response.data, this))
   }
 
+  /**
+   * Get a trip by its characteristics
+   * @param {object} origins_from_station MobilityboxStop or Identifier of the first station of the trip. Usually the start with: `vesputi:station:`
```
Comment (rosario-raulin) [inline src/index.js]: > Usually the start with: `vesputi:station:`

Usually *they* start with? This is typo. The same for the following parameters.

File: src/index.js [hunk 3]
```diff
@@ -68,6 +68,32 @@ export class Mobilitybox {
...
+   * @param {object} destination_station MobilityboxStop or Identifier of the last station of the trip. Usually the start with: `vesputi:station:`
+   * @param {object} origins_from_departure_time The departure time on the starting station as unix timestamp in milliseconds or as MobilityboxEventTime
```
Comment (rosario-raulin) [inline src/index.js]: We should not switch between "origin station" and "start station". By the way: A "starting station" is a strange phrase. Better:

> The departure time of the origin station as a unix timestamp in milliseconds or a MobilityboxEventTime object

File: src/index.js [hunk 4]
```diff
@@ -68,6 +68,32 @@ export class Mobilitybox {
...
+   * @param {object} destination_arrival_time The arrival time on the ending station as unix timestamp in milliseconds or as MobilityboxEventTime
```
Comment (rosario-raulin) [inline src/index.js]: See above. "Ending station" is not a phrase we should use. The right term is "destination station".

File: src/index.js [hunk 5]
```diff
@@ -68,6 +68,32 @@ export class Mobilitybox {
...
+   * @param {String} line_name The short name of the corresponding line
+   * @return {MobilityboxTrip} found trip
```
Comment (rosario-raulin) [inline src/index.js]: The return value is a promise, right? Not the MobilityboxTrip directly. That's confusing then for the user of this API.

File: README.md [hunk 6]
```diff
@@ -109,6 +109,23 @@ const mobilitybox = new Mobilitybox('abc');
 var trip = await mobilitybox.get_trip({id: "vesputi:trip:foobar"})
 ```
 
+#### Mobilitybox.find_trip_by_characteristics({origins_from_station, destination_station, origins_from_departure_time, destination_arrival_time, [line_name]})    
+returns with a *MobilityboxTrip* object based on its meta information.
```
Comment (rosario-raulin) [inline README.md]: Actually it returns a promise that in turn returns a MobilityboxTrip object. We should clarify that.

**Review by rosario-raulin — CHANGES_REQUESTED** [review]
Please fix the documentation issues that I mentioned and then we can merge that.

**Review by rosario-raulin — APPROVED** [review]

**Pull request merged**